wherehows-dao/src/main/java/wherehows/dao/ConnectionPoolProperties.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package wherehows.dao;
+
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Builder;
+
+
+@Builder
+public class ConnectionPoolProperties {
```

Comment:
Rather than building a map, I think it's fine to build EntityManagerFactory directly